Add link to translation and extension (#87)
timdeschryver/timdeschryver.dev: content/blog/making-sure-youre-using-the-correct-query/index.md

![Testing playground suggesting to use another query](./images/testing-playground.png)

> Our use the browser extension for [Chrome](https://chrome.google.com/webstore/detail/testing-playground/hejbmebodbijjdhflfknehhcgaklhano) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/testing-playground/#:~:text=Testing%20Playground%20is%20an%20extension,while%20encouraging%20good%20testing%20practices.)

Reviewer: ```suggestion
Or you can use the browser extension for [Chrome](https://chrome.google.com/webstore/detail/testing-playground/hejbmebodbijjdhflfknehhcgaklhano) or [Firefox](https://addons.mozilla.org/en-US/firefox/addon/testing-playground).
```
